corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -466,8 +466,23 @@ def _migrate_case_actions(couch_case, sql_case):
 
 
 def _migrate_case_attachments(couch_case, sql_case):
-    # TODO: maybe wait until case attachments are in blobdb
-    pass
+    """Copy over attachment meta """
+    attachments = []
+    for name, attachment in six.iteritems(couch_case.case_attachments):
+        attachments.append(CaseAttachmentSQL(
+            name=name,
+            case=sql_case,
+            identifier=attachment.identifier,
+            attachment_src=attachment.attachment_src,
+            attachment_from=attachment.attachment_from,
+            content_type=attachment.server_mime,
+            content_length=attachment.content_length,
+            blob_id=attachment.id,
```

Comment:
I don't think `CommCareCaseAttachment` objects have an `id` property. You'll probably need to get that from `case.blobs[name]`

Reply:
Yup, nice 6eacb96